Session.js

```diff
@@ -0,0 +1,65 @@
+class Session { 
+// To be utilized when unwrapping JSON from the future servlets. Will be on the 
+// client side and be used as a buffer for client to server information needed  
+// for other components of Virtual Movie Night.
+ 
+    constructor() {
+        const expected = {
+            sessionId: 'JTC6',
+            screenNameOfController: 'chris',
+            ipOfVM: '01',
+            listOfAttendees: ['chris', 'bryan']
+        };
+        this.sessionId = expected.sessionId;
+        this.ipOfVM = expected.ipOfVM;
+        this.listOfAttendees = expected.listOfAttendees;
+        this.screenNameOfController = expected.screenNameOfController; 
+    }
+    
+    setSessionId(id) { 
+        this.sessionId = id;
+        return this.sessionId;
```

Comment:
no need to return here, you are just setting the value of the sessionId

Reply:
Yeah setters do not have return values unless you're using a builder pattern, which we are not